### DISCUSSION TITLE
Welcome to ChromeAppHeroes Discussions!

### DISCUSSION THREAD

**Discussion by zhaoolee** [zhaoolee/ChromeAppHeroes#50] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by njxhws** [comment]
how to install to chrome, please

**Comment by liujianpc** [comment]
您好，本人已收到您的邮件！谢谢！      祝您工作顺利，生活愉快！
